docs/components/configMaps.md

```diff
@@ -0,0 +1,146 @@
+<!--
```

Comment:
I suspect that the linting checks were not running, and I was correct.  In your `Gruntfile.js`, you only check files in `docs`, and not anything in a subdirectory.  Please update that check to use `docs/**/*.js` and fix any linting errors.

Reply:
Thanks, @the-t-in-rtf for pointing this out. I have updated it.